tools/runners/VeribleExtractor.py

```diff
@@ -33,16 +33,27 @@ def prepare_run_cb(self, tmp_dir, params):
         inc_dirs = ",".join(params.get("incdirs", []))
 
         with open(script_path, "w") as script:
-            s = (
-                'log="$({executable}'
+            command = (
+                '{executable}'
                 ' --file_list_root ""'
                 ' --include_dir_paths {inc_dirs}'
-                ' --file_list_path {src_list_path}'
-                ' 2>&1 1>/dev/null)"\n'
-                'if [ -n "$log" ]; then echo "$log"; exit 1; fi\n').format(
+                ' --file_list_path {src_list_path}').format(
                     executable=self.executable,
                     inc_dirs=shlex.quote(inc_dirs),
                     src_list_path=shlex.quote(src_list_path))
+            s = (
+                'echo "#" {command_str}\n'
+                'log="$( {command} 2>&1 1>/dev/null )"\n'
+                'rc=$?\n'
+                'echo "stderr:"\n'
+                'echo "$log"\n'
+                'if [ $rc -eq 0 ]; then\n'
+                '    ! echo "$log" | grep -v -e "^I .*] Re-defining macro.*" -e "^$" -q\n'
+                '    rc=$?\n'
+                'fi\n'
+                'exit $rc\n').format(
+                    command_str=shlex.quote(command),
```

Comment:
Looks like the Python [formatter was not happy](https://github.com/SymbiFlow/sv-tests/pull/1270/checks?check_run_id=1688950278#step:6:1) here.

Reply:
Fixed